crates/runc-rust/README.md

```diff
@@ -0,0 +1,8 @@
+# Shim extension for containerd
```

Comment:
Could you create `crates/shim-runc/README.md` too, and explain unimplemented features?